### PR TITLE
Feature/845 jump to listing

### PR DESCRIPTION
**Pull request by chris-lawton**
Based off https://github.com/OpenTechFund/opentech.fund/tree/feature/872-notes-list which in turn is based off https://github.com/OpenTechFund/opentech.fund/tree/feature/875-submit-notes

- adds jump to listing functionality
- moves `<ListingDropdown>` into `<GroupedListing>`

### CODE REVIEW

File: opentech/apply/funds/templates/funds/base_submissions_table.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 {% block extra_js %}
     {{ filter.form.media.js }}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/iamdustan-smoothscroll/0.4.0/smoothscroll.min.js"></script>
```
Comment (todd-dembrey) [inline opentech/apply/funds/templates/funds/base_submissions_table.html]: Is this required still? and do we want it as a lib in package if including?

File: opentech/static_src/src/app/src/components/ListingDropdown/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import PropTypes from 'prop-types';
+
+export default class ListingDropdown extends React.Component {
+    static propTypes = {
+        listRef: PropTypes.object,
+        groups: PropTypes.array,
+    }
+
+    handleChange(e) {
+        const groupHeaderPosition = document.getElementById(e.target.value).offsetTop - 75;
```
Comment (todd-dembrey) [inline opentech/static_src/src/app/src/components/ListingDropdown/index.js]: this 75 is a bit magic...  i'm not sure i'm a fan. There must be a better way?

File: opentech/apply/funds/templates/funds/base_submissions_table.html (hunk 1 above)
Comment (chris-lawton) [reply opentech/apply/funds/templates/funds/base_submissions_table.html]: nope, will remove

Comment (chris-lawton) [reply opentech/apply/funds/templates/funds/base_submissions_table.html]: still required to polyfill `element.scrollTo` in ms browsers, but will move it to be added as a package

File: opentech/static_src/src/app/src/components/ListingDropdown/index.js (hunk 2 above)
Comment (chris-lawton) [reply opentech/static_src/src/app/src/components/ListingDropdown/index.js]: yup, agreed. I'll grab the height of the dropdown container then pass as a prop to `<ListingDropdown>` to use as an offset

**Pull request merged**